File: sgilt-core/src/main/java/net/franzka/sgilt/core/evenement/dto/EvenementSummaryDto.java
package net.franzka.sgilt.core.evenement.dto;

import java.time.LocalDate;
import java.util.UUID;

public record EvenementSummaryDto(
        UUID id,
        String title,
        LocalDate date,
        String ville,
        String imagePath,
        String eventType,
        int confirmedCount,
        int inDiscussionCount
) {}

File: sgilt-front/app/data/evenement/dto/EvenementSummaryDto.ts
/**
 * DTO — contrat de réponse de l'API GET /events (liste)
 */
export interface EvenementSummaryDto {
  id: string
  title: string
  date?: string
  ville?: string
  coverUrl?: string | null
  eventType?: string
  confirmedCount: number
  inDiscussionCount: number
}
